packages/create-plugin/src/utils/utils.npm.ts

```diff
@@ -1,41 +1,16 @@
-import path from 'path';
-import fs from 'fs';
 import semver from 'semver';
-import { readJsonFile } from './utils.files';
-import { renderTemplateFromFile, getTemplateData } from './utils.templates';
-import { GRAFANA_FE_PACKAGES, TEMPLATE_PATHS } from '../constants';
+import { GRAFANA_FE_PACKAGES } from '../constants';
+import { getPackageJson, writePackageJson, getLatestPackageJson } from './utils.packagejson';
 import { PackageManager } from './utils.packageManager';
 
 type UpdateSummary = Record<string, { prev: string | null; next: string | null }>;
 
 type UpdateOptions = {
   onlyOutdated?: Boolean;
   ignoreGrafanaDependencies?: boolean;
+  devOnly?: boolean;
 };
 
-type PackageJson = {
```

Comment:
I had to move these methods out of this file to be able to mock them in jest. It also makes this file smaller